### PR TITLE
Mms enabled refactor

### PR DESCRIPTION
**Pull request by jezek**
Follows PR #6 .
Formats, refactors and adds some logging to new mmsEnabled code.

### CODE REVIEW

**Comment by jezek** [comment]
Now, after merging #6 and rebasing this branch, the PR is only 1 commit long, as intended.

The PR #5 sits atop of this branch (and shares this commit), so I would like to merge this PR before, if it's possible.

Thanks.

File: cmd/nuntium/mediator.go
Comment (dobey) [inline cmd/nuntium/mediator.go]: `MMS` is not plural so this text doesn't read right to me. It should be `is` instead of `are`. Or perhaps something like `mmsEnabled: disabled` instead.

**Review by dobey — CHANGES_REQUESTED** [review]

File: cmd/nuntium/mediator.go
Comment (jezek) [reply cmd/nuntium/mediator.go]: I changed it to "MMS is disabled". The "mmsEnabled: " prefix is for mmsEnabled() function log.

File: cmd/nuntium/mediator.go
Comment (dobey) [inline cmd/nuntium/mediator.go:462]: Why is the return value logic reversed between the if statement above and here, in this commit? Can you also please describe in the PR description how to test this change? Thanks.

**Review by dobey — CHANGES_REQUESTED** [review]

File: cmd/nuntium/mediator.go
Comment (jezek) [reply cmd/nuntium/mediator.go:462]: So it is semantically consistent. See this example:
```go
func someFunc() bool {
  err:= someCommand()
  if err != nil { //early return
    return true
  }

  value, err := someOtherCommand()
  if err == nil {
    return value
  }

  return true
}
```
Like this it's confusing. Best practice is make early return on errors in conditions and value return at end. Like this:
```go
func someFunc() bool {
  err:= someCommand()
  if err != nil { //early return
    return true
  }

  value, err := someOtherCommand()
  if err != nil { //early return
    return true
  }

  return value
}
```

Its logicaly the same, but less confusing. And that's what I've done in this PR. There is really nothing to test (that's why I originally wanted this commit to be part of the previous "merge" PR).
But if someone wants to test it, then the test is the practically the same as for previous "merge" PR #6 . If you disable MMS somewhere in settings, you should not receive MMS sent to you (except there should be "MMS is disabled" in logs).

Comment (dobey) [reply cmd/nuntium/mediator.go:462]: Ah, I see. I missed the `!` change there.

**Review by dobey — APPROVED** [review]

**Pull request merged**
